Argo/Types/Decoded.swift

```diff
@@ -85,3 +85,10 @@ public extension Decoded {
     }
   }
 }
+
+public func ?? <T>(lhs: Decoded<T>, rhs: T) -> T {
```

Comment:
we should stick this where we stuck Alternative `<|>` ... I think in `DecodedOperators.swift`